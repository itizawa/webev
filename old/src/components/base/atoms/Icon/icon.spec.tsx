import renderer from 'react-test-renderer';
import { Icon } from './index';

test('Icon', () => {
  const tree = renderer.create(<Icon icon="ARROW" color="LIGHT" />).toJSON();

  expect(tree).toMatchSnapshot();
});

test('Icon with width and height', () => {
  const tree = renderer.create(<Icon width={32} height={32} icon="ARROW" color="LIGHT" />).toJSON();

  expect(tree).toMatchSnapshot();
});
