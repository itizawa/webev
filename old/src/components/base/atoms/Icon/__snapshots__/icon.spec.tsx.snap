// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon 1`] = `
<svg
  className="align-text-bottom text-light"
  fill="currentColor"
  height={18}
  viewBox="0 0 16 16"
  width={18}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M8 15a.5.5 0 0 0 .5-.5V2.707l3.146 3.147a.5.5 0 0 0 .708-.708l-4-4a.5.5 0 0 0-.708 0l-4 4a.5.5 0 1 0 .708.708L7.5 2.707V14.5a.5.5 0 0 0 .5.5z"
    fillRule="evenodd"
  />
</svg>
`;

exports[`Icon with width and height 1`] = `
<svg
  className="align-text-bottom text-light"
  fill="currentColor"
  height={32}
  viewBox="0 0 16 16"
  width={32}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M8 15a.5.5 0 0 0 .5-.5V2.707l3.146 3.147a.5.5 0 0 0 .708-.708l-4-4a.5.5 0 0 0-.708 0l-4 4a.5.5 0 1 0 .708.708L7.5 2.707V14.5a.5.5 0 0 0 .5.5z"
    fillRule="evenodd"
  />
</svg>
`;
